subworkflows/local/longread_preprocessing.nf

```diff
@@ -17,31 +17,22 @@ workflow LONGREAD_PREPROCESSING {
     ch_multiqc_files = Channel.empty()
 
     if ( !params.longread_qc_skipadaptertrim && params.longread_qc_skipqualityfilter) {
```

Comment:
Missing MultiQC mixing from the adapterrermoval subworkflow

Reply:
Updated!